### ISSUE TITLE
[Bug] App crashes on startup

### ISSUE THREAD

**Issue by blastrock** [siavash79/PixelXpert#826]
### Describe the issue

After I upgraded a few weeks ago, the app would not start anymore

### Steps to reproduce

1. Start the app
2. Crash

### Expected behavior

The app should open

### Your device setup (Please take some time to give correct information here)

- Google Pixel 4a 5g
- Android 14
- October 5, 2023
- Magisk loader 27001, app 27000
- app manager, pixelxpert, shell, systemui, termux
- pixelxpert canary-344, playintegrityfix 15.9.7, systemless hosts 1.0, zygisk - lsposed 1.9.2 (7024)
- LSPosed 1.9.2
- My only LSPosed module is PixelXpert

### Logs

I must confess I did not disable all my Magisk module to reproduce because the bug seems unrelated. Please tell me if you need me to.

```
04-11 19:01:50.186  1305  1305 D Zygote  : Forked child process 13203
04-11 19:01:50.186  1979  2606 I ActivityManager: Start proc 13203:sh.siava.pixelxpert/u0a222 for next-top-activity {sh.siava.pixelxpert/sh.siava.pixelxpert.SplashScreenActivityNormalIcon}
04-11 19:01:50.198 13203 13203 I iava.pixelxpert: Using CollectorTypeCC GC.
04-11 19:01:50.203 13203 13203 E iava.pixelxpert: Not starting debugger since process cannot load the jdwp agent.
04-11 19:01:50.243 13203 13203 W BpBinder: Linking to death on org.lsposed.lspd.service.ILSPApplicationService but there are no threads (yet?) listening to incoming transactions. See ProcessState::startThreadPool and ProcessState::set
04-11 19:01:50.248 13203 13203 I LSPosed : Loading xposed for sh.siava.pixelxpert/10222
04-11 19:01:50.249 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.250 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.250 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.251 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.251 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.252 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.255 13203 13203 I LSPosed-Bridge: Loading legacy module sh.siava.pixelxpert from /system/priv-app/PixelXpert/PixelXpert.apk
04-11 19:01:50.365 13203 13203 W iava.pixelxpert: Unsupported class loader
04-11 19:01:50.367 13203 13203 I LSPosed-Bridge:   Loading class sh.siava.pixelxpert.XPEntry
04-11 19:01:50.374 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.374 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.375 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.375 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.376 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.376 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.402 13203 13203 D CompatibilityChangeReporter: Compat change id reported: 171979766; UID 10222; state: ENABLED
04-11 19:01:50.402 13203 13203 D CompatibilityChangeReporter: Compat change id reported: 242716250; UID 10222; state: ENABLED
04-11 19:01:50.411 13203 13203 W ziparchive: Unable to open '/system/priv-app/PixelXpert/PixelXpert.dm': No such file or directory
04-11 19:01:50.411 13203 13203 W ziparchive: Unable to open '/system/priv-app/PixelXpert/PixelXpert.dm': No such file or directory
04-11 19:01:50.489 13203 13203 D nativeloader: Configuring clns-shared-4 for other apk /system/priv-app/PixelXpert/PixelXpert.apk. target_sdk_version=34, uses_libraries=, library_path=/system/priv-app/PixelXpert/lib/arm64:/system/lib6
04-11 19:01:50.496 13203 13203 W System.err: SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
04-11 19:01:50.496 13203 13203 W System.err: SLF4J: Defaulting to no-operation (NOP) logger implementation
04-11 19:01:50.496 13203 13203 W System.err: SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
04-11 19:01:50.500 13203 13203 W System  : ClassLoader referenced unknown path: 
04-11 19:01:50.510 13203 13203 V GraphicsEnvironment: Currently set values for:
04-11 19:01:50.510 13203 13203 V GraphicsEnvironment:   angle_gl_driver_selection_pkgs=[]
04-11 19:01:50.510 13203 13203 V GraphicsEnvironment:   angle_gl_driver_selection_values=[]
04-11 19:01:50.510 13203 13203 V GraphicsEnvironment: ANGLE GameManagerService for sh.siava.pixelxpert: false
04-11 19:01:50.510 13203 13203 V GraphicsEnvironment: sh.siava.pixelxpert is not listed in per-application setting
04-11 19:01:50.516 13203 13203 D ActivityThread: Loading provider sh.siava.pixelxpert: sh.siava.pixelxpert.utils.RemotePrefProvider
04-11 19:01:50.521 13203 13203 I LSPosed-Bridge: PixelXpert Version: canary-344
04-11 19:01:50.521 13203 13203 I LSPosed-Bridge: PixelXpert Records: 17
04-11 19:01:50.544 13203 13203 D WM-WrkMgrInitializer: Initializing WorkManager with default configuration.
04-11 19:01:50.568 13203 13203 D WM-PackageManagerHelper: Skipping component enablement for androidx.work.impl.background.systemjob.SystemJobService
04-11 19:01:50.568 13203 13203 D WM-Schedulers: Created SystemJobScheduler and enabled SystemJobService
04-11 19:01:50.579  1979  5635 I ActivityManager: Flag disabled. Ignoring finishAttachApplication from uid: 10222. pid: 13203
04-11 19:01:50.597 13203 13222 D CompatibilityChangeReporter: Compat change id reported: 160794467; UID 10222; state: ENABLED
04-11 19:01:50.605 13203 13203 D AppCompatDelegate: Checking for metadata for AppLocalesMetadataHolderService : Service not found
04-11 19:01:50.609 13203 13203 E iava.pixelxpert: Invalid resource ID 0x00000000.
04-11 19:01:50.623 13203 13203 E ThemeUtils: View class androidx.appcompat.widget.AppCompatImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
04-11 19:01:50.629 13203 13203 E iava.pixelxpert: Invalid resource ID 0x00000000.
04-11 19:01:50.629 13203 13203 D AndroidRuntime: Shutting down VM
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: Crash unexpectedly: java.lang.RuntimeException: Unable to start activity ComponentInfo{sh.siava.pixelxpert/sh.siava.pixelxpert.ui.activities.SplashScreenActivity}: android.view.InflateE
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3782)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3922)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.os.Handler.dispatchMessage(Handler.java:106)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.os.Looper.loopOnce(Looper.java:205)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.os.Looper.loop(Looper.java:294)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.ActivityThread.main(ActivityThread.java:8177)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at java.lang.reflect.Method.invoke(Native Method)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: Caused by: android.view.InflateException: Binary XML file line #26 in sh.siava.pixelxpert:layout/activity_splash_screen: Binary XML file line #26 in sh.siava.pixelxpert:layout/activity_
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: Caused by: android.view.InflateException: Binary XML file line #26 in sh.siava.pixelxpert:layout/activity_splash_screen: Error inflating class com.google.android.material.textview.Mater
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: Caused by: java.lang.reflect.InvocationTargetException
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at java.lang.reflect.Constructor.newInstance0(Native Method)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.view.LayoutInflater.createView(LayoutInflater.java:866)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1018)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:973)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1135)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1096)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:694)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at java.lang.reflect.Method.invoke(Native Method)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at waygfPvwTffsoSytSv.FE.bkdry.X.HookBridge.invokeOriginalMethod(Native Method)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at J.callback(Unknown Source:193)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at LSPHooker_.inflate(Unknown Source:21)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:538)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at sh.siava.pixelxpert.databinding.ActivitySplashScreenBinding.inflate(ActivitySplashScreenBinding.java:59)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at sh.siava.pixelxpert.databinding.ActivitySplashScreenBinding.inflate(ActivitySplashScreenBinding.java:53)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at sh.siava.pixelxpert.ui.activities.SplashScreenActivity.onCreate(SplashScreenActivity.java:47)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.Activity.performCreate(Activity.java:8595)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.Activity.performCreate(Activity.java:8573)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3764)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3922)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.os.Handler.dispatchMessage(Handler.java:106)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.os.Looper.loopOnce(Looper.java:205)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.os.Looper.loop(Looper.java:294)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.app.ActivityThread.main(ActivityThread.java:8177)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at java.lang.reflect.Method.invoke(Native Method)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 5: TypedValue{t=0x2/d=0x7f040117 a=-1}, theme={InheritanceMap=[], Themes=[0, forced, sh.siava.pi
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.content.res.TypedArray.getColorStateList(TypedArray.java:600)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at B.ir.boQ.m.R.yEmu.Neykj.ources$XTypedArray.getColorStateList(Unknown Source:84)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.widget.TextView.readTextAppearance(TextView.java:4251)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.widget.TextView.<init>(TextView.java:1249)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at android.widget.TextView.<init>(TextView.java:1149)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at androidx.appcompat.widget.AppCompatTextView.<init>(AppCompatTextView.java:113)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at com.google.android.material.textview.MaterialTextView.<init>(MaterialTextView.java:92)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	at com.google.android.material.textview.MaterialTextView.<init>(MaterialTextView.java:87)
04-11 19:01:50.630 13203 13203 I LSPosed-Bridge: 	... 32 more
04-11 19:01:50.630 13203 13203 E AndroidRuntime: FATAL EXCEPTION: main
04-11 19:01:50.630 13203 13203 E AndroidRuntime: Process: sh.siava.pixelxpert, PID: 13203
04-11 19:01:50.630 13203 13203 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{sh.siava.pixelxpert/sh.siava.pixelxpert.ui.activities.SplashScreenActivity}: android.view.InflateException: Binary XML
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3782)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3922)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:205)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:294)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8177)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: Caused by: android.view.InflateException: Binary XML file line #26 in sh.siava.pixelxpert:layout/activity_splash_screen: Binary XML file line #26 in sh.siava.pixelxpert:layout/activity_
04-11 19:01:50.630 13203 13203 E AndroidRuntime: Caused by: android.view.InflateException: Binary XML file line #26 in sh.siava.pixelxpert:layout/activity_splash_screen: Error inflating class com.google.android.material.textview.Mater
04-11 19:01:50.630 13203 13203 E AndroidRuntime: Caused by: java.lang.reflect.InvocationTargetException
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at java.lang.reflect.Constructor.newInstance0(Native Method)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.view.LayoutInflater.createView(LayoutInflater.java:866)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1018)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:973)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1135)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1096)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:694)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at waygfPvwTffsoSytSv.FE.bkdry.X.HookBridge.invokeOriginalMethod(Native Method)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at J.callback(Unknown Source:193)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at LSPHooker_.inflate(Unknown Source:21)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:538)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at sh.siava.pixelxpert.databinding.ActivitySplashScreenBinding.inflate(ActivitySplashScreenBinding.java:59)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at sh.siava.pixelxpert.databinding.ActivitySplashScreenBinding.inflate(ActivitySplashScreenBinding.java:53)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at sh.siava.pixelxpert.ui.activities.SplashScreenActivity.onCreate(SplashScreenActivity.java:47)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8595)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8573)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3764)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3922)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:205)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:294)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8177)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 5: TypedValue{t=0x2/d=0x7f040117 a=-1}, theme={InheritanceMap=[], Themes=[0, forced, sh.siava.pi
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.content.res.TypedArray.getColorStateList(TypedArray.java:600)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at B.ir.boQ.m.R.yEmu.Neykj.ources$XTypedArray.getColorStateList(Unknown Source:84)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.widget.TextView.readTextAppearance(TextView.java:4251)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.widget.TextView.<init>(TextView.java:1249)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at android.widget.TextView.<init>(TextView.java:1149)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at androidx.appcompat.widget.AppCompatTextView.<init>(AppCompatTextView.java:113)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at com.google.android.material.textview.MaterialTextView.<init>(MaterialTextView.java:92)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	at com.google.android.material.textview.MaterialTextView.<init>(MaterialTextView.java:87)
04-11 19:01:50.630 13203 13203 E AndroidRuntime: 	... 32 more
04-11 19:01:50.639 13203 13203 I Process : Sending signal. PID: 13203 SIG: 9
04-11 19:01:50.649  1979  5839 I ActivityManager: Process sh.siava.pixelxpert (pid 13203) has died: fg  TOP 
04-11 19:01:50.649  1305  1305 I Zygote  : Process 13203 exited due to signal 9 (Killed)
04-11 19:01:50.649  1979  2642 I libprocessgroup: Successfully killed process cgroup uid 10222 pid 13203 in 0ms

**Comment by EleoXDA** [comment]
Install the apk in zip/system/priv-app/PixelXpert folder and it will be fixed

**Comment by blastrock** [comment]
It works now, thanks!

**Comment by EleoXDA** [comment]
> It works now, thanks!

I know. I just hope more and more people started searching in "issues" (closed especially) before creating a new one. This was maybe already answered 15-20 times in already closed issues. This way you would've gotten your answer even faster

**Comment by blastrock** [comment]
Well, I did, as I always do. It was just unfortunate that I didn't find these issues.

Just so that you know how it went, I searched for something like "crash start". The only titles that seemed relevant to me were:

- https://github.com/siavash79/PixelXpert/issues/797
- https://github.com/siavash79/PixelXpert/issues/752

Which don't provide a resolution, but that's to be expected with such generic search terms. I then searched for "Failed to resolve attribute" which I assume is the most specific error in the log dump, and found only these two same issues. At least now if someone searches for the error string they will find this issue :)

I know it's tough to maintain such a big project, and do it for free, but unfortunately having reporting already-resolved issues is just part of it. I think it's fine to take a break from answering issues if you just don't feel like it. We (or at least I) are still grateful for the work you put in this tool!
